src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.SslProvider.cs

```diff
@@ -180,8 +205,30 @@ private static int VerifyCertChain(IntPtr storeCtxPtr, IntPtr arg)
 
                         using (X509Certificate2 leafCert = new X509Certificate2(leafCertPtr))
```

Comment:
@stephentoub I'm having trouble following the lines of code that you are referring to in WinHttpHandler.  Can you clarify which lines of code in WinHttpHandler have this problem?